src/Main.elm

```diff
@@ -25,40 +143,939 @@ init =
+
+        Phone ->
+            "Phone"
+
+        Post ->
+            "Post"
+
+
+fromIdToString : Ids -> String
+fromIdToString ids =
+    case ids of
+        PrepId ->
+            "preparation"
+
+        CloseId ->
+            "closed"
+
+        CancelId ->
+            "canceled"
+
+        TimeOutId ->
+            "timedOut"
+
+        RejectId ->
+            "rejected"
+
+        ErrId ->
+            "error"
+
+        EmailId ->
+            "email"
+
+        PhoneId ->
+            "phone"
+
+        CompanyId ->
+            "companyName"
+
+        PostId ->
+            "address"
+
+        ZipId ->
+            "zip"
+
+        CityId ->
+            "city"
+
+        NewTagNameId ->
+            "newTagName"
+
+        NewTagValId ->
+            "newTagValue"
+
+        CountryId ->
+            "countryName"
+
+
+
+---- VIEW ----
 
 
 view : Model -> Html Msg
-view _ =
-    Html.div [ Attrs.class "flex flex-col w-[1024px] items-center mx-auto mt-16 mb-48" ]
-        [ header "Let's start your task"
-        , subheader "Here are your data:"
-        , Html.pre [ Attrs.class "my-8 py-4 px-12 text-sm bg-slate-100 font-mono shadow rounded" ] [ Html.text Data.userGroup ]
-        , header "Now turn them into form."
-        , subheader "See README for details of the task. Good luck 🍀 "
+view model =
+    Html.div [ Attr.class "flex flex-col sm:w-full md:w-[626px] items-center mx-auto" ]
+        [ case model.userGroup of
+            Initial ->
+                text ""
+
+            Loading ->
+                Html.div []
+                    [ Html.span [ Attr.class "loader" ] []
+                    ]
+
+            Error error ->
+                Html.div [ Attr.class "flex items-center z-12 bg-red-200 mt-10 border px-3 py-2 mx-4 rounded border-red-400 text-2xl text-red-400 " ]
+                    [ Html.span [ Attr.class "w-[30px] flex mr-2" ] [ Icons.cautionIcon ]
+                    , Html.p [] [ text error ]
+                    ]
+
+            Success userGroup ->
+                Html.div [ Attr.class "flex flex-col mx-4 md:mx-0 mt-16 mb-48" ]
+                    [ Html.div [ Attr.class "mx-4 md:mx-0" ]
+                        [ Html.div [ Attr.class "flex w-[140px] h-[30px]" ]
+                            [ Html.img [ Attr.src "logo.png" ] []
+                            ]
+                        , Html.h1 [ Attr.class "text-3xl  md:text-5xl mb-10" ] [ text "User Group Settings Form" ]
+                        ]
+                    , if model.isFormSent then
+                        Html.div [ Attr.class "flex mb-4 text-green-400 rounded border border-green-400 bg-green-200 px-3 py-2" ]
+                            [ Html.span [ Attr.class "mr-2 flex w-[20px]" ] [ Icons.checkIcon ]
+                            , Html.p [] [ text "Success ! Form has been sent !" ]
+                            ]
+
+                      else
+                        text ""
+                    , Html.div [ Attr.class "" ]
```

Comment:
Q: what is the meaning of having `Attr.class ""`?

Reply:
It was leftover from  previous attempt to style that element, then eventually styled parent or child, I think there are at least 2 more occasions where i forgot to delete unused Attr.class ""